target/linux/ath79/dts/ar9132_tl_wr1043nd_v1.dts

```diff
@@ -22,19 +26,18 @@
 	};
 
 	keys {
-		compatible = "gpio-keys-polled";
+		compatible = "gpio-keys";
```

Comment:
why change it to real keys ?

Reply:
Saw somewhere that this is deprecated, they do work tho